app/controllers/content_item_signups_controller.rb

```diff
@@ -43,13 +43,11 @@ def handle_redirects
     destination_path = @content_item.dig("redirects", 0, "destination")
     return error_not_found if destination_path.nil?
 
-    redirect_to(new_content_item_signup_path(topic: destination_path))
+    redirect_to(new_content_item_signup_path(link: destination_path))
   end
 
   def assign_content_item
-    # Topic param left in for backwards compatibility.
-    # Topic is the user-facing terminology for taxons. Expect the taxon base
-    # path to be provided in a param of this name.
+    # Note: the "topic" param has historically appeared in external links
```

Comment:
That's interesting. Do we get direct visits to signup pages from external sites?

Reply:
Yup. I don't actually know if such links make use of the "topic" query param, but I'm airing on the side of caution in case we change how taxon signup works in future, when it might be tempting to remove it here - whoever is making the change should go and check. I did also mean that our other apps use "topic". Once we've switched to "link", it should be much clearer in future analytics/logs whether this query param is actually used externally.